packages/daimo-api/src/api/createRequestSponsored.ts

```diff
@@ -40,5 +46,10 @@ export async function createRequestSponsored(
     args: [id, recipient, BigInt(amount), metadata],
   });
 
+  // Store memo, if applicable
+  if (memo != null) {
+    paymentMemoTracker.addMemo(requestTxHash, memo);
```

Comment:
one slight ugliness: request memos are tracked by *tx hash*, not op hash

i think this is OK:
* txhash and ophash are non-overlapping
* for permissionless request creation, it'd be an op anyway
* it's currently a tx because we want to allow people to send requests without onchain auth = without FaceID

(separately, session key auth in the API will make that setup less resistant to abuse. coming soon. cc @kayleegeorge )